### PR TITLE
#9722: Sandbox Physics AR tests 

### PR DESCRIPTION
**Pull request by FuzzyCarterAWS**
**Notes**:
Let's make sure we're getting @o3de/sig-simulation & @o3de/sig-testing stakeholders to approve this before merging.

**Description**
Disabling Physics AR tests while tests are undergoing refactor/reimplementation.
* Moved tests in TestSuite_Main to TestSuite_Sandbox
* Removed TestSuite_Main.py
* Removed CTest TestSuite_Main reference from CMakeLists.txt

Please see #9722 for detailed information

Signed-off-by: FuzzyCarterAWS <jscar@amazon.com>

**Testing Done:**
Verified that the Physics AR tests no longer runs through local testing via Pytest & Ctest calls.

### CODE REVIEW

**Review by AMZN-Dk — APPROVED** [review]
Agreed on direction of disabling these tests and rebuilding them.

File: AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py [hunk 1]
```diff
@@ -14,7 +14,9 @@
 from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorBatchedTest, EditorTestSuite
 from .utils.FileManagement import FileManagement as fm
 
+
 # Custom test spec, it provides functionality to override files
+@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Phyiscs AR Tests")
```
Comment (lumberyard-employee-dm) [inline AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py:19]: Typo "Phyiscs"
```suggestion
@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Physics AR Tests")
```

File: AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py [hunk 2]
```diff
@@ -53,6 +55,7 @@ def wrap_run(cls, instance, request, workspace, editor, editor_test_results, lau
         for f in original_file_list:
             fm._restore_file(f, file_list[f])
 
+@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Phyiscs AR Tests")
```
Comment (lumberyard-employee-dm) [inline AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py:58]: Repeat Type: "Phyiscs"
```suggestion
@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Physics AR Tests")
```

**Review by lumberyard-employee-dm — COMMENTED** [review]
This test can also be disabled by removing the pytest registration with CTest.

**Comment by hultonha** [comment]
Agree with @greerdv's comments

**Comment by hultonha** [comment]
Also for a little more context, we have discussed this on the Physics team and we agreed disabling the Physics automated tests is the right thing to do right now.

We're going to schedule some work to audit all existing tests. The plan is to delete any redundant or outdated tests to get to a more sane number. The plan will then be to take one at a time, look at updating it, run it 100 times on a test machine under heavy load, and then and only then, add it back to the main test rotation.

We'll then wait around a week before adding a new test to make sure there are no additional failures (that are not legitimate). This will be happening over the next several weeks/months. Please reach out to me if you have any other questions.

**Comment by Kadino** [comment]
I'm comfortable with the spirit of this change, though the tests should be disabled to the preference of SIG-Simulation.

**Review by smurly — APPROVED** [review]

**Review by smurly — APPROVED** [review]

**Review by greerdv — APPROVED** [review]

**Review by moraaar — APPROVED** [review]

**Review by hultonha — APPROVED** [review]

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**